### DISCUSSION TITLE
Return Selected Best Model

### DISCUSSION THREAD

**Discussion by Chrisischris** [open-meteo/open-meteo#1063] [Ideas]
When making a request to the forecast api using "best_match" for the models parameter, would it be possible to include the model that was selected in the response for each location?

**Comment by patrick-zippenfenig** [comment]
Hi,  
Yes, this will be included in the future. It's a bit more complex than it might seem, as the API integrates various weather models with different forecast lengths, weather parameters, and also uses internal models to generate extra parameters. In practice, each hour, each time-step, and each location could potentially come from a different source.
